data/targets.yml

```diff
@@ -59,11 +59,21 @@
   obs_project: "Devel:YaST:SLE-15-SP2"
   obs_sr_project: "SUSE:SLE-15-SP2:Update"
   obs_target: "SUSE_SLE-15-SP2_GA"
+:sle15sp3:
+  obs_api: "https://api.suse.de/"
+  obs_project: "Devel:YaST:SLE-15-SP3"
+  obs_sr_project: "SUSE:SLE-15-SP3:Update"
+  obs_target: "SUSE_SLE-15-SP3_GA"
+:sle15sp4:
+  obs_api: "https://api.suse.de/"
+  obs_project: "Devel:YaST:SLE-15-SP4"
+  obs_sr_project: "SUSE:SLE-15-SP4:Update"
```

Comment:
this looks strange, if you intentionally build against SLE15 SP4 it will use Update channel. Why not GA?

Reply:
It the same as the other targets, for SLE15-SP4 GA we will use `sle_latest`.